### PR TITLE
Real Python: elections.

### PR DESCRIPTION
**Pull request by dgilboa**
Add elections.

### CODE REVIEW

File: realpython/ch07/election.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Simulation of elections
```
[the remark this answers is not in the document]
Comment (dgilboa) [reply realpython/ch07/election.py]: 👍 Already done in later exercises (-:
will fix here as well.

File: realpython/ch07/election.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+regions_normalized = [
+    {'A': .87, 'B': 1},
+    {'A': .65, 'B': 1},
+    {'A': .17, 'B': 1}
+]
+
+
+def simulate_a_region(region):
+    winner = random()
+    for candidate, norm_odd in region.iteritems():
```
[the remark this answers is not in the document]
Comment (dgilboa) [reply realpython/ch07/election.py]: you're right ! we'll fix.

[the remark this answers is not in the document]
Comment (dgilboa) [reply realpython/ch07/election.py]: after some reading - orderedDict is not suitable as it only preserves the order of insert and does not keep the dict real ordered by a specific order, so for now I'll sort by the candidate odd value in the normalised probability map.

**Comment by dlahyani** [comment]
@dgilboa you can merge.

**Pull request merged**